ferretdb/os_unix.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
+// +build !windows
+
+package ferretdb
+
+import (
+	"context"
+	"os/signal"
+
+	"golang.org/x/sys/unix"
+)
+
+// notifyAppTermination is for unix-related context notify.
+func notifyAppTermination(parent context.Context) (context.Context, context.CancelFunc) {
+	return signal.NotifyContext(parent, unix.SIGTERM, unix.SIGINT)
```

Comment:
The library package should not install signal handlers

Reply:
> The library package should not install signal handlers

removed.